src/components/checkbox/checkbox-panel/checkbox-panel.stories.ts

```diff
@@ -0,0 +1,180 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { InputType } from '@storybook/types';
+import type { Args, ArgTypes, Decorator, Meta, StoryObj } from '@storybook/web-components';
+import { html, type TemplateResult } from 'lit';
+
+import { sbbSpread } from '../../../storybook/helpers/spread.js';
+
+import '../../button.js';
+import '../../card.js';
+import '../../icon.js';
+import './checkbox-panel.js';
+
+import readme from './readme.md?raw';
+
+const checked: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const indeterminate: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const disabled: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const label: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const value: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const name: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const ariaLabel: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const defaultArgTypes: ArgTypes = {
+  checked,
+  indeterminate,
+  disabled,
+  label,
+  value,
+  name,
+  'aria-label': ariaLabel,
+};
+
+const defaultArgs: Args = {
+  checked: false,
+  indeterminate: false,
+  disabled: false,
+  label: 'Label',
+  value: 'Value',
+  name: 'name',
+  'aria-label': undefined,
+};
+
+const Template = ({ label, checked, ...args }: Args): TemplateResult =>
+  html` <sbb-checkbox-panel .checked=${checked} ?checked=${checked} ${sbbSpread(args)}>
```

Comment:
Why the duplicate checked prop (? and .)?

Reply:
I think it's needed to make the checkbox behave just like the native one, Jeri and Lukas mentioned the fact that they needed to jump through some hoops to achieve that